doc/release/new-policy.rst

```diff
@@ -222,22 +223,22 @@ The stage splits into two phases:
 Support
 ~~~~~~~
 
-The stage starts when a first release is published.
+The stage starts when the first release is published.
 The release series now is an object of only backward compatible changes.
 
 At this stage, all known security problems and all found
 degradations since the previous series are fixed.
 
-A series receives degradation fixes and other bugfixes during the support stage
+The series receives degradation fixes and other bugfixes during the support stage
 and until the series transitions into the end of life (EOL) stage.
 
 The decision of whether to fix a particular problem in a particular release series
-depends on the impact of the problem, risks around backward compatibility and the
+depends on the impact of the problem, risks around backward compatibility, and the
 complexity of backporting a fix.
 
-A release series might receive new features at this stage,
+The release series might receive new features at this stage,
 but only in a backward compatible manner.
-A release candidate might be published for a new functionality before a release.
+A release candidate might be published if new functionality is introduced before the release version.
```

Comment:
```suggestion
   To collect feedback for new functionality, we may also publish a release candidate before the final release version.
```

Reply:
I would say 'next' instead of 'final'.

Reply:
```suggestion
   We may also publish a release candidate to collect feedback before the release version.
```
I looked at the context more closely and this looks like a better fit. What do you think?

Reply:
Looks ok.